src/autocomplete/EmojiProvider.tsx

```diff
@@ -75,9 +78,18 @@ export default class EmojiProvider extends AutocompleteProvider {
     matcher: QueryMatcher<ISortedEmoji>;
     nameMatcher: QueryMatcher<ISortedEmoji>;
     private readonly recentlyUsed: IEmoji[];
-
+    emotes: Dictionary<any>;
+    emotesPromise: Promise<any>;
     constructor(room: Room, renderingType?: TimelineRenderingType) {
         super({ commandRegex: EMOJI_REGEX, renderingType });
+        const emotesEvent = room?.currentState.getStateEvents("m.room.emotes", "");
```

Comment:
We can't use `m.room.emotes` until the MSC is accepted - until then, we use an `UnstableValue`